media/CircleEditor/index.js

```diff
@@ -146,6 +150,12 @@ host.BrowserHost = class {
                 case 'reload':
                     this._msgReload(message);
                     break;
+                case 'edit':
+                    this._msgLoadModel(message);
+                    break;
+                case 'customType':
+                    this._msgGetType(message);
```

Comment:
It's hard to think about the meaning of this function :(
What is this function for?

Reply:
For attributes of the custom operator, the type is not recognized by netron. So it's a function that takes the type from the model and fills in the attributes type of the custom operator.

Reply:
As there are so many kinds of "type", it is really confused :(
For example,
- `customType` : Is there any type like this? Which type in custom operator, attributes' or inputs' or outputs'?
- `msgGetType` : Which type can be gotten? All operators or only some of operators?

Could you clarify the names?

Reply:
There could be some confusion... `msgGetCustomOperatorAttributesType` I think it will be a little long to express them all clearly, do you have any good ideas?

Reply:
Hmm... how about `msgGetCustomOpAttrT`?
It's little bit longer too but shorter than full name...

Reply:
I'd like that. I'll apply it! Thank you!